framework/test/unit/modules/interoperability/internal_method.spec.ts

```diff
@@ -1373,6 +1601,34 @@ describe('Base interoperability internal method', () => {
 				expect.toBeObject() as Certificate,
 			);
 		});
+
+		it('should resolve correctly when validators store is NOT sorted', async () => {
+			jest.spyOn(cryptography.bls, 'verifyWeightedAggSig').mockReturnValue(true);
+
+			await interopMod.stores
+				.get(ChainValidatorsStore)
+				.set(methodContext, txParams.sendingChainID, {
+					...chainValidators,
+					activeValidators: [...activeValidators].sort(() => Math.random() - 0.5), // Shuffle activeValidators
```

Comment:
There are only 4 entries in the list. Hence, once in 24 executions (on average) the list is sorted correctly. Could you easily change the order such that it is ensured that it is not ordered? Like swapping the first and second entry.

Reply:
Updated in a non-hardcoded way [here](https://github.com/LiskHQ/lisk-sdk/blob/4cbdadaf90ceccfe5fdf242f688ee3d30da90c4d/framework/test/unit/modules/interoperability/internal_method.spec.ts#L1623C8-L1623C23)